contracts/DateTime.sol

```diff
@@ -214,7 +215,7 @@ contract DateTime {
         }
 
         function __throw() {
-                uint[] arst;
+                uint[] memory arst;
```

Comment:
If you're willing to, can you just remove this function and replace any invocations of it with the `throw` statement?  This code was written before `throw` existed.